app/src/js/components/Datepicker/Datepicker.js

```diff
@@ -173,32 +163,32 @@ export class Datepicker extends React.PureComponent {
         <div className="datetime">
           <div className="datetime__range">
             <ul className="datetime__internal">
-                  <li>
-                    { this.renderDateRangeDropDown() }
-                  </li>
-                  <li>
-                    { this.renderDateTimeRange('startDateTime') }
-                  </li>
-                  <li>
-                  <span> to </span>
-                  </li>
-                  <li>
-                    { this.renderDateTimeRange('endDateTime') }
-                  </li>
-                  <li className="selector__hrformat">
-                    { this.renderHourFormatSelect() }
-                  </li>
-                </ul>
-              </div>
-            <div className="datetime__wrapper">
+              <li>
+                { this.renderDateRangeDropDown() }
+              </li>
+              <li data-cy='startDateTime'>
+                { this.renderDateTimeRange('startDateTime') }
+              </li>
+              <li>
+                <span> to </span>
+              </li>
+              <li data-cy='endDateTime'>
+                { this.renderDateTimeRange('endDateTime') }
```

Comment:
Is there a plan to add validation if the end date is before the start date?

Reply:
I wasn't going to, and the thought occurred to me, we could handle it in a number of ways.  I thought I'd see what people liked and didn't like about the functionality. If we need that for this PR, I can add it.